### PR TITLE
Change mappings to be compatible with ES6

### PR DESCRIPTION
**Pull request by hannayurkevich**
Changes include:
1) Replace datatype "string" with "keyword" for not analyzed values and "text" for full-text-search values (as per https://www.elastic.co/blog/strings-are-dead-long-live-strings). Use "index":true for all values.
2) Change mappings to define properties in one type "doc" instead of multiple types (as per https://www.elastic.co/guide/en/elasticsearch/reference/6.x/removal-of-types.html). Note the name is chosen to correspond the name used in mappings in elasticsearch-boshrelease es6 branch ( see commit https://github.com/cloudfoundry-community/logsearch-boshrelease/commit/1c48c2fce1eaa5e5e49aeedc48a1b8d852203261)

### CODE REVIEW

File: jobs/elasticsearch-config-lfc/templates/index-mappings-app.json.erb [hunk 1]
```diff
@@ -1,82 +1,82 @@
...
 # This file contains mappings specific for 'app' indicies.
 #
-# By default all string fields are set as `not_analyzed`.
+# By default all string fields are set as `keyword` (not analyzed values).
 #
 
 require 'json'
 
-string_default = { "type": "string", "index": "not_analyzed" }.to_json
+keyword_default = { "type": "string", "index": true }.to_json
```
Comment (axelaris) [inline jobs/elasticsearch-config-lfc/templates/index-mappings-app.json.erb]: It seems to be a typo. `keyword` is expected

File: jobs/elasticsearch-config-lfc/templates/index-mappings-app.json.erb [hunk 2]
```diff
@@ -87,25 +87,21 @@ string_default = { "type": "string", "index": "not_analyzed" }.to_json
...
           }
-        }
-      }
-    },
-
-    "ContainerMetric": {
-      "properties": {
+        },
+        
         "@cf": {
```
Comment (axelaris) [inline jobs/elasticsearch-config-lfc/templates/index-mappings-app.json.erb]: It's a second instance of `@cf`. Probably added by mistake

**Review by axelaris — CHANGES_REQUESTED** [review]
Could you please do two amendments, and point this PR to `es6` branch instead of `develop`.
Thanks.

**Review by axelaris — APPROVED** [review]

**Comment by hannayurkevich** [comment]
@axelaris 

I've made requested changes. Please check.

**Comment by axelaris** [comment]
Thank you @hannayurkevich!

**Pull request merged**